README.md

```diff
@@ -1,71 +1,86 @@
+<img align="right" src="images/logo.svg" alt="SimpleX logo" height="90">
+
 # SimpleX chat
 
+## Federated, private, secure, decentralized
+
 [![GitHub build](https://github.com/simplex-chat/simplex-chat/workflows/build/badge.svg)](https://github.com/simplex-chat/simplex-chat/actions?query=workflow%3Abuild)
 [![GitHub release](https://img.shields.io/github/v/release/simplex-chat/simplex-chat)](https://github.com/simplex-chat/simplex-chat/releases)
 
-## Federated chat - private, secure, decentralized
+SimpleX chat objective: to give you the full control of your chat with free open-source protocol and software of simple to deploy servers and simple to use clients providing complete privacy, security and ownership of your contacts and chat.
```

Comment:
I suggest removing this line - coming to a repo (e.g. from a website, or friend's recommendation) you usually already know what you're generally looking for, and if not the second line refers to the site which repeats and expands on this, so I feel like it only gets in the way of real information